iot-hub/Samples/service/ImportExportDevicesSample/Parameters.cs

```diff
@@ -6,51 +6,59 @@ namespace Microsoft.Azure.Devices.Samples
     /// <summary>
     /// Configurable parameters for the sample.
     /// </summary>
+    /// <remarks>
+    /// To get these connection strings, log into https://azure.portal.com, go to Resources, open the IoT hub, open Shared Access Policies, open iothubowner, and copy a connection string.
+    /// </remarks>
     internal class Parameters
     {
         [Option(
             'i',
             "SourceIoTHubConnectionString",
-            Required = true,
-            HelpText = "The service connection string with permissions to manage devices for the source IoT hub to copy devices. Log into https://azure.portal.com, go to Resources, open the IoT hub, open Shared Access Policies, open iothubowner, and copy a connection string.")]
+            Required = false,
+            HelpText = "The service connection string with permissions to manage devices for the source IoT hub to copy devices. "
+                + "Defaults to environment variable 'SOURCE_IOTHUB_CONN_STRING_CSHARP'.")]
```

Comment:
Using apostrophes instead of quotation marks to avoid required escaping them (readability).